\documentclass[10pt,twocolumn,letterpaper]{article}

\usepackage{cvpr}
\usepackage{times}
\usepackage{epsfig}
\usepackage{graphicx}
\usepackage{amsmath}
\usepackage{amssymb}
\usepackage{enumerate}

% Include other packages here, before hyperref.

% If you comment hyperref and then uncomment it, you should delete
% egpaper.aux before re-running latex.  (Or just hit 'q' on the first latex
% run, let it finish, and you should be clear).
\usepackage[pagebackref=true,breaklinks=true,letterpaper=true,colorlinks,bookmarks=false]{hyperref}

\cvprfinalcopy % *** Uncomment this line for the final submission

\def\cvprPaperID{****} % *** Enter the CVPR Paper ID here
\def\httilde{\mbox{\tt\raisebox{-.5ex}{\symbol{126}}}}

% Pages are numbered in submission mode, and unnumbered in camera-ready
\ifcvprfinal\pagestyle{empty}\fi
\begin{document}

%%%%%%%%% TITLE
\title{Anomalies Detection In DNA Sequences Using Markov Chains}

\author{Pierette M. Mastel \quad Pamely Zantou \quad Florent C. Bang Njenjock \quad Cedric P. E. Manouan \\
Carnegie Mellon University\\
{\tt\small \{pmahoro, pzantou, fbangnje, cmanouan\}@andrew.cmu.edu}
}

\maketitle
\thispagestyle{empty}

%%%%%%%%% ABSTRACT
\begin{abstract}
    Many genetic disorders are rooted in mis-sequencing the
    genome in DNA \cite{posey}. Detecting anomalies in genomic sequences, finding the right genomic code, and reconstructing
    defective sequences represent great challenges and subjects
    of important and expensive research work in medical care.
    Many revolutionary approaches in genomic medicine, bioinformatics, and mathematics have been developed in biological
    sequence analysis to minimize and even completely cure
    genetic disorders. This work aims to model DNA Sequences
    using discrete-time Markov chains and apply the resulting model to detect Cystic fibrosis. 
\end{abstract}

%%%%%%%%% BODY TEXT
%%\section{Introduction}
%%{
%%}

%-------------------------------------------------------------------------
\section{Background and rationale}
 {
    \par Cystic fibrosis (CF) is a severe autosomal recessive disease characterized by a varied degree of chronic obstructive lung disease and pancreatic
    enzyme insufficiency. An elevated sweat electrolyte concentration is another symptoms of (CF) \cite{riordan1989identification}\cite{tsui1992spectrum}.
    CF is caused by the mutation of Cystic Fibrosis Transmembrane conductance Regulator (CFTR) gene. More specifically, a 3bp deletion of CFTR would remove 
    the phenylalanine residue at amino acid position 508. Although, the major mutation causing CF is the later, there are almost 300 sequence alterations 
    identified in the CFTR and responsible of CF. \cite{tsui1992spectrum}. Large deletions spanning multiple exons of the CFTR, multiple substitutions of 
    a single base pair can be found in many places in the gene.

    \begin{itemize}
   
   \item \textbf{DNA sequence models}
   \par{
       DNA sequences analysis has been and continues to be a challenging topic in the research domain. However, to be able to do these analyzes, one needs a way to represent the DNA sequence into a suitable format 
       that helps to extract information and do some computation.
       \begin{enumerate}[i.]
           \item \textbf{Probabilistic approaches to DNA sequence modeling}
           \begin{enumerate}[a.]
               \item \textbf{The Independent Identically Distributed (IID) model}: This model assigns the same chance to each nucleotide to appear at any position within the sequence (uniformly likely).
               In addition, each nucleotide appears independently of the others. The probability of any nucleotide \(X\) to occur in a sequence \(S\) of length \(L_S\) is defined as following (G. Singh et al. 2003):
               \begin{equation}
                   P_X = \frac{n_{X}(S)}{L_S}\\
                   \label{eq1}
               \end{equation}
               where \(n_X\) is the number of occurrences of the base X in S.\\
               Then the probability of a pattern \(p\) to occur can be obtained with equation \ref{eq2}
               \begin{equation}
                    P(p | S) = \prod_{i=1}^{L_p} P(p_i)
                   \label{eq2}
               \end{equation}
               where \(P(p_i)\) is the probability of nucleotide \(p_i\) at position \(i\) along a pattern p of length \(L_p\).\\
               
               \item \textbf{Discrete-time Markov chains}: In this model, the value of the random variable X any given time step depends only on the previous value.
               This model has 20 parameters: the probabilities of each nucleotide to occur (\(P_A, P_T, P_C, P_G\)) and the probabilities of each \textit{dinucleotide} to occur (\(P_{AA}, P_{AC}, ...\)).
               The first set of parameters is computed using equation \ref{eq1}.
               And the second set of parameters (the transition probabilities) is obtained \cite{singh} using Bayes Rule (\ref{eq3}):\\
               \begin{equation}
                   P(\alpha|\beta) = \frac{P(\alpha \beta)}{P(\beta)}
                \label{eq3}
               \end{equation}
               where \(\alpha\) and \(\beta\) are different nucleotides and \(\alpha \beta \) is a dinucleotide (a compound with two nucleotides).\\ 
               Thus the probability of occurrence of a pattern p is given by equation \ref{eq4}:
               \begin{equation}
                   P(p|S) = P(p_1)\prod_{i=2}^{L_p} P(p_i)
                \label{eq4}
               \end{equation}
               \end{enumerate}
       \end{enumerate}
   }
    \end{itemize}
 }

 %-------------------------------------------------------------------------
\section{Literature review}
{

Mutations in gene may lead to severe diseases. In the case of Cystic Fibrosis (CF), studies were conduced and revealed that some Single Nucleotide Polymorphisms (SNPs) may be associated with presence of chronic rhinosinusitis (CRS) in certain populations, which is almost always developed by CF patients  \cite[Bejamen Hull et al., 2017]{SNP_cystic_fibrosis}. Identifying those mutations quickly is crucial since they might have a high influence on disease severity in different manners \cite[Franziska Gisler, 2013]{SNP_identification}. 
Thus, strategies for newborn screening for CF have been evaluated and as a result, it was shown that immunioreactive trypsinogen (IRT), a test that screens for a protein made by the pancreas, seems to be the most cost-efficient among others \cite[Masja Schmidt et al., 2018]{Newborn_screening_for_CF}. In case of a positive initial IRT test, another test or a DNA is carried out. This DNA test can lead to the identification of modifier genes (genes than single mutated gene) which affect disease expression.
And the most common method to study association between genetic variations in modifier genes and clinical phenotypes is SNPs identification \cite[Martijn Slieker et al., 2005]{disease_modifying_genes}. 


}
%-------------------------------------------------------------------------

%-------------------------------------------------------------------------
\section{Research questions, aims and objectives}
 {
    Our main research question is: \textit{how can we model DNA sequences using Markov chains in order 
    to use the resulting representation to detect disorders/anomalies in a given DNA sequence?}\\
    Trying to answer the above question boils down to two problems that need to be addressed:
    \begin{itemize}
        \item How to model DNA sequences using Markov chains?
        \item How to use this model to detect anomalies in DNA sequences \cite{scientific_american}?
    \end{itemize}

 }
%-------------------------------------------------------------------------

\section{Methodology}
 {  
    \subsection{Modeling}  
    {
        The overall goal of our work is to be able to identify specific patterns within a DNA sequence in order to tell whether or not that sequence has some anomalies.\\
        For the purpose of this research, we will be using discrete-time Markov chains as DNA sequences model \cite{singh}. 
       Our model will be based on the four nucleotides (A, T, G, C) which will be considered as the states of the system.
       This approach to modeling DNA sequences is divided into two parts: on the one hand, we will work on representing nucleotides and the relationships between them in a given DNA sequence (graph of states); 
           and on the other hand, we will compute the transition probabilities within the resulting graph to end up with the probabilistic graph representing the chain.
    }  

    \subsection{Anomaly detection}
    {In this work, an \textit{anomaly} is any nucleotides sequence that is known as the sequence of a particular disease. Since this pattern must occur at a precise location of the DNA sequence to be considered as an actual 
    anomaly, we are considering using a threshold before returning the binary detection output.\\ 
    To achieve anomalies detection given a DNA sequence we plan to use the following steps:
    \begin{enumerate}[a.]
        \item Compute the first set of parameters (individual nucleotides)
        \item Compute the second set of parameters (dinucleotides)
        \item Select anomaly pattern to match
        \item Get the probabilities of possible matches at a given position of the sequence
        \item Apply a threshold to the resulting probabilities
        \item Return a binary signal (Anomaly detected or not)
    \end{enumerate}
    }
 }

 \section{Dataset}
 {
    \href{https://www.ncbi.nlm.nih.gov/genbank/}{NCBI} will be our main data source. Through NCBI, we have access to the GenBank genetic sequence database,
     an annotated collection of all publicly available DNA sequences.
     As mentioned, Cystic Fibrosis is caused by CFTR mutations and through NCBI  we have access to several CFTR gene sequences that carry a specific CFTR mutation.
 }
%-------------------------------------------------------------------------

%\section{Results}
 %{
 %}
%-------------------------------------------------------------------------

 %\section{Discussion}
 %{
 %}
%-------------------------------------------------------------------------
%\section{Conclusion}
%{
%}

 {\small
  \bibliographystyle{ieee}
  \bibliography{egbib}
 }

\end{document}